src/pydisagg/ihme/splitter.py

```diff
@@ -74,6 +53,18 @@ def remove_prefix(self) -> None:
             setattr(self, field, getattr(self, field).removeprefix(self.prefix))
 
 
+class DataConfig(BaseModel):
+    index: list[str]
+    age_lwr: str
+    age_upr: str
```

Comment:
If we want to make this a sex splitting specific thing, we should have a new class and remove `age_lwr` and `age_upr`.